packages/components/src/components/NFT/NFT.tsx

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
-import { Box, Heading, Image, Flex, Tag, Text } from '@chakra-ui/react';
+import React, { useCallback, useEffect, useRef } from 'react';
+import { Box, Heading, Image, Flex, Tag, Text, VStack } from '@chakra-ui/react';
+import fetch from 'cross-fetch';
 
 export interface NFTProps {
+  contractAddress: string;
   tokenId: string;
-  name: string | null;
-  imageUrl: string;
-  assetContractName: string;
+}
+
+interface NFTData {
+  tokenId: string;
+  imageUrl?: string;
+  name: string;
```

Comment:
not every NFT has a name. that's why this was nullable and i had `const displayName = name || tokenId;` below

Reply:
Gotcha